### DISCUSSION TITLE
[MOCK IDEA] Support Mercurial repositories for projects

### DISCUSSION THREAD

**Discussion by Humbedooh** [apache/infrastructure-ideas#1] [New Ideas for Services]
This is a sample mock idea to show what suggestions could look like.
Discuss!

**Comment by Humbedooh** [comment]
Upvotes can be used as a sign of support for an idea, signifying a wider potential need for a certain feature in the community.

**Comment by cottage14** [comment]
I take it there is no downvote option. I have found that useful in other systems.

**Reply by cottage14** [reply]
"Sample mock" is a bit redundant. Just sayin'

**Comment by ShaneCurcuru** [comment]
We should add CVS support too!  Would be useful for historical file access from the ASF's early days! 😺

**Comment by dave2wave** [comment]
See this link - https://genius.com/Carly-simon-and-james-taylor-mockingbird-lyrics
